packages/cli/src/cli.ts

```diff
@@ -53,14 +53,33 @@ export async function cli(args: string[] = process.argv) {
           .demandCommand();
       },
       handler: (_args) => {},
+    })
+    .fail(async (msg, err, yargs) => {
+      const Consola = await import("consola");
+      if (err instanceof ExitProcessError) {
+        Consola.consola.error(err);
+      } else {
+        // if yargs.check returns a string it populates the err variable with a string
+        if (err && typeof err !== "string") {
```

Comment:
This is maybe not so obvious to be more explicit we could define a similar custom error class for custom check/validation errors?

Reply:
Yeah I suggested that above. I agree that would be the cleanest will do it